Additions/UIView-KIFAdditions.m

```diff
@@ -248,31 +256,29 @@ - (UIAccessibilityElement *)accessibilityElementMatchingBlock:(BOOL(^)(UIAccessi
                     if ([indexPathsForVisibleRows containsObject:indexPath]) {
                         continue;
                     }
-                    
+
                     @autoreleasepool {
-                        // Get the cell directly from the dataSource because UITableView will only vend visible cells
-                        UITableViewCell *cell = [tableView.dataSource tableView:tableView cellForRowAtIndexPath:indexPath];
-                        
+                        // Scroll to the cell
+                        [tableView scrollToRowAtIndexPath:indexPath atScrollPosition:UITableViewScrollPositionNone animated:NO];
+
+                        UITableViewCell *cell = [tableView cellForRowAtIndexPath:indexPath];
                         UIAccessibilityElement *element = [cell accessibilityElementMatchingBlock:matchBlock notHidden:NO];
-                        
-                        // Remove the cell from the table view so that it doesn't stick around
-                        [cell removeFromSuperview];
```

Comment:
This isn't necessary or caused some problem?

Reply:
It seemed to my iOS developer colleague that it wasn't necessary.